### DISCUSSION TITLE
Read files inside chart

### DISCUSSION THREAD

**Discussion by hypery2k** [helmfile/helmfile#314] [Questions]
Is it somehow possible to read from an existing Helm Chart other value files, e.g. `values.dev.yaml` ?

**Comment by yxxhero** [comment]
@hypery2k Can you show a example?

**Comment by hypery2k** [comment]
Sure.

Let assume we build our apps, bundle the image and publish a helm chart with the following content:
```
Chart.yaml
values.yaml
values.dev.yaml
values.prod.yaml
```

We have several apps and want to use to deploy the different app releases (charts) in their individual version into one helmfile release package and then would like to reuse/extract the included `values.prod.yaml` from each chart

**Comment by yxxhero** [comment]
@hypery2k Maybe you should use `environments`.

**Comment by yxxhero** [comment]
https://helmfile.readthedocs.io/en/latest/#configuration

**Reply by hypery2k** [reply]
we're already using environments for other use cases, but it would be great the keep the `values.<env>.yaml` in each app

Is there a way to get the path of the chart archive to extract it from there?

**Comment by yxxhero** [comment]
@hypery2k I thinks you can copy it into `environments` dir, I don't understand how this demand is generated?

**Reply by hypery2k** [reply]
we want to be able to deploy each app chart individual but also make one "overall" product release via helmfile and therefore want to reuse the already used `values.<env>.yaml` inside the chart
